src/components/ProfilePage/Rewards/RewardsAmountBox/RewardsAmountBox.tsx

```diff
@@ -33,59 +31,55 @@ export const RewardsAmountBox = ({
   );
 
   //CONST
-  const REWARD_CHAIN_LOGO =
-    theme.palette.mode === 'dark'
-      ? PROFILE_CAMPAIGN_DARK_CHAIN
-      : PROFILE_CAMPAIGN_LIGHT_CHAIN;
-  const REWARD_TOKEN_LOGO =
-    theme.palette.mode === 'dark'
-      ? PROFILE_CAMPAIGN_DARK_TOKEN
-      : PROFILE_CAMPAIGN_LIGHT_TOKEN;
+  const REWARD_CHAIN_LOGO = chainLogo; // theme.palette.mode === 'dark' ?
+  const REWARD_TOKEN_LOGO = tokenLogo; // theme.palette.mode === 'dark' ?
 
   return (
-    <FlexCenterRowBox>
+    <FlexCenterRowBox minWidth="96px">
       <Box marginLeft="32px">
         <Image
           src={REWARD_TOKEN_LOGO}
           alt="token image"
-          width={64}
-          height={64}
+          width={40}
+          height={40}
           style={{
             borderRadius: 16,
           }}
         />
         {isMobile ? undefined : (
           <Image
             src={REWARD_CHAIN_LOGO}
-            alt="token image"
-            width={32}
-            height={32}
+            alt="chain image"
```

Comment:
Can we grab the name of the chain for the IMG alt text? To make it more descriptive?